api/staff-ref-data/real-api.ts

```diff
@@ -44,13 +44,20 @@ export async function getJobTitles(req, res: Response, next: NextFunction) {
 
 export async function getServices(req, res: Response, next: NextFunction) {
   const apiPath: string = `${baseCaseWorkerRefUrl}/refdata/case-worker/skill`;
-
   try {
     const { status, data }: { status: number, data } = await handleGet(apiPath, req, next);
 
     const options: StaffFilterOption[] = [];
+
+    const serviceRefData = getConfigValue(SERVICE_REF_DATA_MAPPING) as any;
```

Comment:
Instead of any can you create a type for it? Thanks

Reply:
Updated.